docs/guides/stay-up-to-date/releases/2026-08/index.md

```
@@ -0,0 +1,420 @@
+
+
+### [v6.1.3](https://github.com/gruntwork-io/terraform-aws-architecture-catalog/releases/tag/v6.1.3)
+
+<p style={{marginTop: "-20px", marginBottom: "10px"}}>
+  <small>Published: 8/4/2026 | <a href="https://github.com/gruntwork-io/terraform-aws-architecture-catalog/releases/tag/v6.1.3">Release notes</a></small>
+</p>
+
+<div style={{"overflow":"hidden","textOverflow":"ellipsis","display":"-webkit-box","WebkitLineClamp":10,"lineClamp":10,"WebkitBoxOrient":"vertical"}}>
+
+  * Fix: Update include in service quotas to have a name by @odgrim in https://github.com/gruntwork-io/terraform-aws-architecture-catalog/pull/1181
+* LIB-4203 updating pipelines plan role with eks read only perms by @x-nightwing in https://github.com/gruntwork-io/terraform-aws-architecture-catalog/pull/1217
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)